### ISSUE TITLE
GraphQL plugin breaks if you have a content type named "map"

### ISSUE THREAD

**Issue by EloB** [strapi/strapi#4470]
**Describe the bug**
GraphQL server breaks if you create a content type with name `map`. This will prevent any query sent to the server and return below error.
```json
{
  "error": {
    "errors": [
      {
        "message": "Cannot read property 'maxAge' of undefined",
        "extensions": {
          "code": "INTERNAL_SERVER_ERROR",
          "exception": {
            "stacktrace": [
              "TypeError: Cannot read property 'maxAge' of undefined",
              "    at CacheControlExtension.computeOverallCachePolicy (/Users/ollebroms/Documents/my-project/node_modules/apollo-cache-control/dist/index.js:85:22)",
              "    at Object.<anonymous> (/Users/ollebroms/Documents/my-project/node_modules/apollo-server-core/dist/requestPipeline.js:174:79)",
              "    at Generator.next (<anonymous>)",
              "    at fulfilled (/Users/ollebroms/Documents/my-project/node_modules/apollo-server-core/dist/requestPipeline.js:5:58)"
            ]
          }
        }
      }
    ]
  }
}
```

**Steps to reproduce the behavior**
1. Install a clean install of Strapi
2. Create content type and name it `map`. Create a string field and name it `title`.
3. Install official strapi Graphql plugin.
4. Run any graphql query to replicate error in the Graphql playground.

**Code snippets**
If applicable, add code samples to help explain your problem.

**System**
- Node.js version: v10.16.2
- NPM version:
- Strapi version: 3.0.0-beta.17.5
- Database:
- Operating system:

**Comment by lauriejim** [comment]
Hello! Thank you for reporting this issue.
I suggest you investigate in the GraphQL plugin to see what part of code break this issue.

Check out the contributing guide to get started: https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md

**Comment by lol2x** [comment]
I believe this is related to #2828 which was supposed to be fixed in #4227 but it wasn't
